packages/web-components/src/components/va-textarea/va-textarea.tsx

```diff
@@ -168,6 +179,11 @@ export class VaTextarea {
     return this.maxlength;
   }
 
+  private getHeaderLevel() {
```

Comment:
Can this be imported from the utilities?

`import { getHeaderLevel } from '../../utils/utils';`